figanos/matplotlib/plot.py

```diff
@@ -505,10 +507,14 @@ def gridmap(
                 transform = get_rotpole(data)
 
     # setup fig, ax
-    if ax is None:
+    if ax is None and ("row" not in plot_kw.keys() and "col" not in plot_kw.keys()):
         fig, ax = plt.subplots(subplot_kw={"projection": projection}, **fig_kw)
```

Comment:
With gridmap? Weird it used to work fine for me, but maybe I modified something while working on scattermap

Reply:
Oh yeah, since we are not creating a matplotlib figure when using col/row (xarray itself creates the facetgrid) the only direct fig kwargs passed to the function seems to be figsize. I can see if there's a a work around to send it to the wrapped matplotlib function. Can you share what are the fig_kw you're trying to use?

Reply:
figsize was not working when I wrote this, but it seems to work now.